projects/media-viewer/src/lib/toolbar/right-pane/right-pane.component.html

```diff
@@ -44,4 +44,9 @@
       <span data-l10n-id="tools_label">Tools</span>
     </button>
   </ng-container>
+  <button *ngIf="enableAnnotations && !icpEnabled"
```

Comment:
I think this might still be needed for ICP as well. You can't create annotations/comments, but you can see them